app/api/orders.py

```diff
@@ -766,3 +766,30 @@ def process_transaction(order_identifier, txn_token):
 
     response = PaytmPaymentsManager.hit_paytm_endpoint(url=url, head=head, body=body)
     return response
+
+
+@order_misc_routes.route('/orders/<string:order_identifier>/paytm/transaction-status', methods=['GET'])
+def get_transaction_status(order_identifier):
+    paytm_params = dict()
+    paytm_checksum_params = dict()
+    url = ""
```

Comment:
Include a check alluding to the target endpoint(based on `paytm_mode` in settings)
_reference_:
Staging:
https://securegw-stage.paytm.in/order/status

Production:
https://securegw.paytm.in/order/status

Reply:
I've already used it on line 789 @mrsaicharan1

Reply:
@shreyanshdwivedi I guess that's fine then. Remove this explicit declaration of `url=""` and this is GTG. :D